src/modules/systemlib/circuit_breaker_params.c

```diff
@@ -106,7 +105,7 @@ PX4_PARAM_DEFINE_INT32(CBRK_AIRSPD_CHK);
  * @max 121212
  * @group Circuit Breaker
  */
-PX4_PARAM_DEFINE_INT32(CBRK_FLIGHTTERM);
+PARAM_DEFINE_INT32(CBRK_FLIGHTTERM, 121212);
```

Comment:
The defines in circuit_breaker_params.h were not used in any source I could find in the respository. The only difference with PX4_PARAM_DEFINE_* is that it references those defaults in that header instead of specifyig right in the macro like PARAM_DEFINE_*